utils/make-pkgconfig.swift

```diff
@@ -58,7 +58,7 @@ func makeFile() throws {
   let cllvmPath = pkgConfigDir.appendingPathComponent("cllvm.pc")
 
   /// Ensure we have llvm-config in the PATH
-  guard let llvmConfig = which("llvm-config") else {
+  guard let llvmConfig = which("llvm-config-3.9") ?? which("llvm-config") else {
```

Comment:
I got the opposite impression from [this comment](https://github.com/trill-lang/LLVMSwift/pull/47#issuecomment-274296002), so I'm just double checking